docs/openeuropa-components.md

```diff
@@ -5,97 +5,263 @@ Each project complies with [PHP-FIG][1] standards and adhere to the best-practic
-
-### Authentication track
-
-Maintainers: [Hernani](https://github.com/hernani), [Imanol](https://github.com/imanoleguskiza)
-
-Components:
-- [OpenEuropa Authentication](https://github.com/openeuropa/oe_authentication)
-
-### Search track
-
-Maintainers: [Francesco](https://github.com/brummbar), [Sergii](https://github.com/sergepavle), [Victor](https://github.com/voidtek)
-
-Components:
-- [OpenEuropa Search](https://github.com/openeuropa/oe_search)
-
-### Tooling track
-
-Maintainers: [Antonio](https://github.com/ademarco), [Diogo](https://github.com/dxvargas), [Victor](https://github.com/voidtek)
-
-Components:
-- [Behat transformation context](https://github.com/openeuropa/behat-transformation-context)
-- [Code Review](https://github.com/openeuropa/code-review)
-- [Composer artifacts](https://github.com/openeuropa/composer-artifacts)
-- [Drupal Module Template](https://github.com/openeuropa/drupal-module-template)
-- [Drupal Site Template](https://github.com/openeuropa/drupal-site-template)
-- [Drupal core require dev](https://github.com/openeuropa/drupal-core-require-dev)
-- [OpenEuropa Profile](https://github.com/openeuropa/oe_profile)
-- [Task Runner](https://github.com/openeuropa/task-runner)
-
-## Additional components
-
-Below additional components maintained by other DIGIT teams:
-
-- [Webtools geocoding provider](https://github.com/openeuropa/webtools-geocoding-provider)
-
+### Drupal projects
+
+<table>
+    <tr>
+        <th>Component</th>
+        <th>Status</th>
+    </tr>
+    <tr>
+        <td>
+            <a title="Repository" href="https://github.com/openeuropa/oe_authentication"><b>OpenEuropa Authentication</b></a><br/>
+        This module allows to authenticate users against the European Commission login service. Details of this service can be found <a href="https://webgate.ec.europa.eu/cas/about.html">here</a>.
+        EU Login service parameters are already set by default when installing the module. Please refer to the EU Login documentation for the available
+        options that can be specified. You can see Project setup section on how to override these parameters.</td>
+        <td width="250">
+            <a title="Version" href="https://packagist.org/packages/openeuropa/oe_authentication"><img src="https://img.shields.io/packagist/v/openeuropa/oe_authentication.svg"/></a>
+            <a title="Build" href="https://drone.fpfis.eu/openeuropa/oe_authentication"><img src="https://drone.fpfis.eu/api/badges/openeuropa/oe_authentication/status.svg"/></a>
+            <a title="Downloads" href="https://packagist.org/packages/openeuropa/oe_authentication"><img src="https://img.shields.io/packagist/dt/openeuropa/oe_authentication.svg"/></a>
+        </td></tr>
+    <tr>
+        <td>
+            <a title="Repository" href="https://github.com/openeuropa/oe_content"><b>OpenEuropa Content</b></a><br/>
+    This is a Drupal module that contains the EC corporate entity types.
+    Current content types are:
```

Comment:
I would not list them here. As we add them we have to maintain tis list and most likely we will forget. The details will be on the component repo. The same principle for all components. I would not list here things that we may be adding/removing but more of a general description that can withstand the test of time a bit easier.

Reply:
Yep agreed, we just need to add what the component does, then it's up to them to check each component's readme.